includes/klarna-order-management-functions.php

```diff
@@ -9,6 +9,38 @@
 	exit;
 }
 
+use Automattic\WooCommerce\Internal\DataStores\Orders\CustomOrdersTableController;
+
+/**
+ * Equivalent to WP's get_the_ID() with HPOS support.
+ *
+ * @return int the order ID or false.
+ */
+//phpcs:ignore
```

Comment:
What is this ignore for? Can you add a more specific rule for it to ignore with a description of why?